polar_route/data_loaders.py

```diff
@@ -782,27 +787,13 @@ def load_north_sea_currents(params, long_min, long_max, lat_min, lat_max, time_s
 
                 lat | long | time | uC | vC
     """
-
+    logging.debug("opening file {}".format(params['file']))
     bc = xr.open_dataset(params['file'])
     bc_df = bc.to_dataframe()
 
     bc_df = bc_df.reset_index()
     bc_df = bc_df[['lat', 'lon', 'U', 'V']]
     bc_df = bc_df.rename(columns={'lon': 'long', 'lat': 'lat', 'U': 'uC', 'V': 'vC'})
 
+    logging.debug("returning {} datapoints".format(len(bc_df.index)))
     return bc_df
-
-
-
-def load_oras5(params, long_min, long_max, lat_min, lat_max, time_start, time_end):
```

Comment:
This shouldn't have been removed. This means that you haven't updated from main before submitting a pull request. At this time I am going to reject the pull request until this change is made